### ISSUE TITLE
Ready usable classifier

### ISSUE THREAD

**Issue by ktugan** [BioroboticsLab/bb_tracking#1]
We should upload a readily trained model to a server so easy and fast classifications are possible. 

```python
> from bb_tracking import track

> repo = Repository('2015-09-15')
> tracks = track(repo)
--- model not ready, downloading from http://biorobotics-server.com/bb_tracking_v0.14_model.gzip ---
> len(tracks)
1337
```

Most people dont need to know what bb_tracking does and only want tracks.

**Comment by b2m** [comment]
## Where

We already have several GitHub Repositories with pretrained models for certain tasks:

 - https://github.com/BioroboticsLab/bb_decoder_models
 - https://github.com/BioroboticsLab/deeplocalizer_models
 - https://github.com/BioroboticsLab/bb_pipeline_models

So I guess we should add another repository with model*s* for the tracking process.

## What

At the moment **bb_tracking** only supports testing features for one stage in a tracking pipeline. Examples for stages in a tracking pipeline could be:

1. Merging **Detections** to **Fragments**
1. Anomaly detection for **Detections**
1. Merging **Fragments** to bigger **Fragments**
1. Anomaly detection for **Fragments**
1. continue with 3 until satisfied

For a practical run on the whole data set we not only need several pre-trained classifiers (one for each type of stage), but also a *Scheduler* that distributes the data/time ranges to several *Walker* instances. And afterwards collects the Fragments from each *Walker* and distributes them in the next step...

**My point**: it is not only about **one** model, but about a process that still is in development.
